views/peeps/list.erb

```diff
@@ -0,0 +1,9 @@
+<% if @user %>
+    <h2>Welcome, <%= @user.email %>!</h2>
+<% end %>
+
+<% @peeps.each do |peep| %>
+    <%= peep.date %>
+    <%= peep.time %>
+    <%= peep.message %><br>
+<% end %>
```

Comment:
You could think about adding the peep form to the list page so the user can see the list of peeps whilst creating a new one.